Carbon/Carbon.psd1

```diff
@@ -211,6 +211,7 @@ All functions are idempotent: when run multiple times with the same arguments, y
                             'Install-WindowsFeature',
                             'Invoke-AppCmd',
                             'Invoke-PowerShell',
+                            'Join-CommandLine',

```

Comment:
Let's not expose these functions. Once they are public, we're committed to maintaining them and preserving backwards-compatability. Remove `Join-CommandLine` and `Split-CommandLine` from this list.